SayTheirNames/Source/Utils/Image/UIImage+Resource.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  UIIcons+Resource.swift
+//  SayTheirNames
+//
+//  Created by Visal Rajapakse on 6/5/20.
+//  Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
+//
+
+import UIKit
+
+//collection of icons used
+extension UIImage{
+    enum STN {
+        static let bookmark: UIImage = UIImage(named: KIconNames.bookmarkLiteral) ?? #imageLiteral(resourceName: "bookmark_active")
+        static let bookmarkActive: UIImage = UIImage(named: KIconNames.bookmarkActiveLiteral) ?? #imageLiteral(resourceName: "bookmark_active")
+        static let heartActive: UIImage = UIImage(named: KIconNames.heartActiveLiteral) ?? #imageLiteral(resourceName: "heart_active")
+        static let petitionActive: UIImage = UIImage(named: KIconNames.petitionActiveLiteral) ?? #imageLiteral(resourceName: "petition_active")
+        static let settingsActive: UIImage = UIImage(named: KIconNames.settingsActiveLiteral) ?? #imageLiteral(resourceName: "settings_active")
+        static let whiteBookmark: UIImage = UIImage(named: KIconNames.whiteBookmarkLiteral) ?? #imageLiteral(resourceName: "white-bookmark")
+        static let whiteSearch: UIImage = UIImage(named: KIconNames.whiteSearchLiteral) ?? #imageLiteral(resourceName: "Simple Search Icon")
+        static let imagePlaceHolder: UIImage = UIImage(named: KIconNames.imagePlaceholderLiteral) ?? #imageLiteral(resourceName: "image-placeholder")
+        static let manInRedJacket: UIImage = UIImage(named: KIconNames.manInRedJacketLiteral) ?? #imageLiteral(resourceName: "man-in-red-jacket-1681010")
+    }
+}
+
+// MARK: - Constants
+struct KIconNames {
```

Comment:
Can we make this more inline with `STN` prefix.

Reply:
(or) maybe we should make this an extension of `String` and have something like `String.image.imageName`

Reply:
As @bertadevant suggested below. Let's use `SwiftGen` to handle the image names!